Update Single DC Multi K8S doc (#13278)
hashicorp/consul: website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx

~> **Note:** This deployment topology requires that your Kubernetes clusters have a flat network
for both pods and nodes, so that pods or nodes from one cluster can connect
to pods or nodes in another. In many hosted Kubernetes environments, you will need to explicitly configure your hosting providers network. See [Azure AKS CNI](https://docs.microsoft.com/en-us/azure/aks/concepts-network#azure-cni-advanced-networking), [AWS EKS CNI](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) and [GKE VPC-native clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips).
If a flat network is not available across all Kubernetes clusters, follow the instructions for using [Admin Partitions](/docs/enterprise/admin-partitions), which is a Consul Enterprise feature.

> 

Author: I will add a new requirements section.

> Now we can switch to the second Kubernetes cluster where we will deploy only the Consul clients

Reviewer: ```suggestion
Switch to the second Kubernetes cluster where you will deploy the Consul clients
```
Every instance of "we" should either be "you" or revised into an imperative statement with an implied "you". Per our style guide, use "we" when referring to HashiCorp. We do not use the plural first person (the royal we).

Author: Thank you, @trujillo-adam, for the review. I will incorporate the changes you have proposed.

that will join the first Consul cluster.

```shell-session
$ kubectl config use-context <CONTEXT_NAME>
```

> First, we need to apply credentials we've extracted from the first cluster to the second cluster:

Reviewer: ```suggestion
First, apply the credentials you extracted from the first cluster to the second cluster:
```
